demo/gulpfile.js

```diff
@@ -17,7 +17,9 @@ const proxy = require('express-http-proxy');
 const url = require('url');
 const bodyParser = require('body-parser');
 
-const api_url = process.env.API_URL;
+const apiUrl = process.env.API_URL;
+const demoPort = process.env.DEMO_PORT || '8080';
```

Comment:
perhaps this port should be unified across environments, no need to support more than one default (`4000` vs `8080`)

Reply:
One consideration may be easy development. If it's the same port during development that docker compose exposes than it will clash (but the demo website needs the API). The port could be mapped to a different port in docker compose but that might make it less obvious.